lib/rack/response.rb

```diff
@@ -56,6 +61,12 @@ def initialize(body = nil, status = 200, header = {})
       yield self if block_given?
     end
 
+    attr_accessor :length, :status, :body
+    attr_reader :headers
+
+    # @deprecated Use {#headers} instead.
+    alias header headers
```

Comment:
I think this all should place above, as it was, and as Ruby Style (Community) Guide suggest here: https://rubystyle.guide/#consistent-classes

Reply:
Okay, if that's how it's being handled, I'll investigate & revert that change. I always like to have the `#initialize` first, just from the POV that it defines the shape of the class in a pretty major way.

Reply:
Reverted.

Reply:
Maybe community want to separate state and behavior in classes. And `attr`s are more like state, like constants, and `initialize` is a method as behavior. I'm not sure. Or just instance-method-to-instance-method, and (like-)class-method to class-method.